resources/outputs.ts

```diff
@@ -189,6 +189,14 @@ export default {
     cn: '分散',
     ko: '산개',
   },
+  protean: {
+    en: 'Protean',
+    de: 'Himmelsrichtungen',
+    fr: 'Positions',
+    ja: '基本散会',
+    cn: '八方分散',
+    ko: '8방향 산개',
+  },
```

Comment:
For future reference (and based on the comment by SJoshua), should additions to `outputs.ts` only contain `en` strings in their initial implementation, with a comment indicating general usage, and allow translators to translate them?

Seems like there's a lot of potential for mis-translation if these are copy/pasted from existing mechs which might have mech-specific hints in the translated text.

Reply:
Yeah that's probably a good idea.